pkg/config/controller_config.go

```diff
@@ -82,6 +84,12 @@ func configInit(sess *session.Session, metadata EC2Metadata) error {
 		ServiceNetworkOverrideMode = true
 	}
 
+	privateVPC := os.Getenv(DISABLE_TAGGING_SERVICE_API)
```

Comment:
Please update `privateVPC` variable name to `disableTaggingServiceAPI` or similar

Reply:
oops, didn't realize I missed this. Updated.